### ISSUE TITLE
Cannot add the TextNG 7.0 library

### ISSUE THREAD

**Issue by vkadlcik** [cbeust/testng-eclipse#445]
### Problem Statement

I cannot create a project with TestNG 7.0.0 or add the library to an existing project. I always get Invalid classpath container: 'TestNG'.

My SW:
- Fedora 30
- OpenJDK 1.8 (java-1.8.0-openjdk-devel-1.8.0.212.b04-0.fc30.x86_64)
- Eclipse Platform + JDT of SimRel 2019-06
- org.testng 7.0.0.r201908191551
- org.testng.eclipse 7.0.0.201908201535

[configuration.txt](https://github.com/cbeust/testng-eclipse/files/3533857/configuration.txt) is my Eclipse configuration taken from Help > About. [setup.sh.txt](https://github.com/cbeust/testng-eclipse/files/3533870/setup.sh.txt) is my minimal reproducing setup script (note: it's destructive).

Steps to reproduce the problem:
1. File > New > Project... > Java > Java Project, click "Next >"
2. Project name: o.e.vk.p1, click "Next >"
3. Select the Libraries tab, click "Add Library...", double-click "TextNG", click "Finish", click "Finish"
4. Confirm switching to the Java perspective

Immediately you get 2 errors in the Problems view, the primary one is:
  Invalid classpath container: 'TestNG' in project 'o.e.vk.p1'
![screenshot1](https://user-images.githubusercontent.com/54436400/63582812-0f123e00-c59a-11e9-88ba-3b986e90e71e.jpg)
The workspace log says:
```!ENTRY org.eclipse.jdt.ui 4 10001 2019-08-23 10:59:38.197
!MESSAGE Null entry in container 'org.testng.TESTNG_CONTAINER'

!ENTRY org.eclipse.ui 4 0 2019-08-23 11:00:54.486
!MESSAGE Unhandled event loop exception
!STACK 0
java.lang.NullPointerException
        at org.eclipse.jdt.internal.ui.packageview.ClassPathContainer.getChildren(ClassPathContainer.java:202)
        ...
```

Now on exiting Eclipse, you get a window with "Problems saving workspace"
![screenshot2](https://user-images.githubusercontent.com/54436400/63582813-0faad480-c59a-11e9-971b-a0673ed73426.jpg)
and in the workspace log:
```!ENTRY org.eclipse.core.resources 4 2 2019-08-23 11:04:12.036
!MESSAGE Problems occurred when invoking code from plug-in: "org.eclipse.core.resources".
!STACK 0
java.lang.NullPointerException
        at org.eclipse.jdt.internal.compiler.util.HashtableOfObjectToInt.get(HashtableOfObjectToInt.java:75)
        ...
```

Here is the [workspace log](https://github.com/cbeust/testng-eclipse/files/3533858/workspace.log) in full.

I see similarities with #443 and #444 so it's possible that all these issues share a common root cause.

**Comment by missedone** [comment]
thanks for reporting,
fixed in new release 7.0.0.201908240652
pls verify

**Comment by vkadlcik** [comment]
> thanks for reporting,
> fixed in new release 7.0.0.201908240652

pretty swift :-)

> pls verify

works for me, thanks for fixing this
